src/parse.c

```diff
@@ -2097,6 +2097,7 @@ handle_routes(NetplanParser* npp, yaml_node_t* node, __unused const void* _, GEr
 
 err:
     route_clear(&npp->current.route);
+    npp->current.route = NULL;
```

Comment:
comment: FWIW, I think this nullification is already being done, using the `CLEAR_FROM_FREE` macro. But I guess it doesn't hurt to do it manually again in the error case.

Reply:
hmm yeah, that's true. Coverity (and I) wasn't smart enough to see that I guess... well, if nothing, this line will eliminate an issue from the Coverity report.